release-packaging/HelpSource/Classes/FluidDataSetWr.schelp

```diff
@@ -67,58 +67,38 @@ s.reboot;
 (
 ~ds.clear;
 OSCFunc({
-    "FluidDataSetWr help: all points written".postln;
-    ~ds.print
+	"FluidDataSetWr help: all points written".postln;
+	~ds.print
 },'/datasetwrdone').oneShot;
 
-{  |n|
-	var b = LocalBuf.newFrom([0,1,2,3]);
-    var trig = Impulse.kr(ControlRate.ir / 8);
-    var idx = Stepper.kr(trig,min:-1, max:n); //we need to start at -1 to catch the first increment
-	4.collect{|i| BufWr.kr([(4 * idx) + i],b,i)};
-    FluidDataSetWr.kr(~ds,idNumber:idx,buf:b,trig:trig);
-    SendReply.kr(idx >= (n-1), '/datasetwrdone');
-    FreeSelf.kr(idx >= (n-1));
-}.play(s,args:[n:100]);
+{
+	arg n;
+	var buf = LocalBuf(4);
+	var trig = Impulse.kr(ControlRate.ir / 8); // can't go any faster
```

Comment:
Why can't we go any faster? Is this empirical or is there a logical explanation (I would have thought `ControlRate.ir / 2` would be more likely)